neo/Network/P2P/Payloads/ConsensusPayload.cs

```diff
@@ -15,12 +16,28 @@ public class ConsensusPayload : IInventory
         public UInt256 PrevHash;
         public uint BlockIndex;
         public ushort ValidatorIndex;
-        public uint Timestamp;
```

Comment:
Actually, the `PrepareRequest` already was going to break the format since it removes the signature. I think the only change we need to make is be able to still forward the existing consensus messages from previous clients. Once CN nodes are updated old clients will be required to upgrade, which is probably reasonable, but we should make these changes stay compatible so they can be run on seed nodes with the existing consensus. In this way seed nodes can upgrade and run this code before CN nodes are updated to the new dBFT 2.0 consensus mechanism.

Reply:
I'm working on a PR that will allow the code to still work in the existing network.

Reply:
Please review #609.

Reply:
I will do that now. I also have #610 that I just made.

Reply:
I think we need to do a bit more to ensure we can properly forward the `ConsensusPayload` in the existing network.  Can you look at #610.

Reply:
Fix merged.